docs/containers/quickstart-python.md

```diff
@@ -0,0 +1,142 @@
+
+7. With all this information, the Docker extension creates the following files:
+
+    - A `Dockerfile`. To learn more about Intellisense in this file, refer to the [overview](/docs/containers/overview.md).
+
+    - A `.dockerignore` file to reduce the image size by excluding files and folders that aren't needed such as `.git`, `.vscode`, and `__pycache__`.
+
+    - If Docker Compose was selected, a `docker-compose.yml` file.
+
+    - If one does not already exist, a `requirements.txt` file for capturing all app dependencies.
+    > **Tip**: If the virtual environment/host machine is meant to be identical to the container, ensure app dependencies are ported over by running `pip freeze > requirements.txt` in the terminal to overwrite this file.
+
+### File modifications for Django/Flask apps
+
+To give Python Web Developers a great starting point, we chose to use [Gunicorn](https://gunicorn.org/#docs) as the default web server. Since it is referenced in the default Dockerfile, it is included as a dependency in the `requirements.txt` file.
+
+> **Note**: To use Gunicorn as your web server, it must be included in the `requirements.txt` file as an app dependency. It does not need to be installed in your virtual environment/host machine.
+
+#### Django Apps
+
+To use Gunicorn, it must bind to an application callable as an entry point. This callable is declared in the `wsgi.py` file of a Django application. To accomplish this binding, the final line in the Dockerfile says:
+
+```docker
+CMD ["gunicorn", "--bind", "0.0.0.0:8000", "{workspace_folder_name}.wsgi"]`
+```
+
+This assumption is based off Django's default project structure. If your project does not follow `django_project_name (workspace folder) -> django_project_name (subfolder) -> wsgi.py`, you must overwrite the Gunicorn entry point in the Dockerfile to locate the correct `wsgi.py` file.
+
+> **Tip**: If your `wsgi.py` file is in the root folder, the final argument in the command above will be `"wsgi"`. Within subfolders, the argument would be `"subfolder1_name.subfolder2_name.wsgi"`.
+
+#### Flask Apps
+
+To use Gunicorn, it must bind to an application callable as an entry point. This callable corresponds with the **file location** and **variable name** of your created Flask instance. According to [official Flask Documentation](https://flask.palletsprojects.com/en/1.1.x/api/), users generally create a Flask instance in the main module or in the `__init__.py` file of their package in this manner:
+
+```python
+from flask import Flask
+app = Flask(__name__) # Flask instance named app
+```
+
+To accomplish this binding, the final line in the Dockerfile says:
+
+```docker
+CMD ["gunicorn", "--bind", "0.0.0.0:5000", "{subfolder}.{module_file}:app"]
+```
+
+You have already given the path to the Flask instance, but the Docker extension assumes your Flask instance variable is named `app`. If this is not the case, the variable name must be overwritten in the Dockerfile.
```

Comment:
I'd rewrite the passive voice in the second sentence here - do you mean "you must overwrite the variable name" or does the tool do something -it's ambiguous.

Reply:
Changed to "If this is not the case, you must change the variable name in the Dockerfile."